packages/optimizer/lib/fetchRuntimeParameters.js

```diff
@@ -34,27 +39,123 @@ async function fetchRuntimeParameters(config, customRuntimeParameters) {
   const runtimeParameters = Object.assign({}, customRuntimeParameters);
   // Configure the log level
   runtimeParameters.verbose = customRuntimeParameters.verbose || config.verbose || false;
+  await initValidatorRules(runtimeParameters, customRuntimeParameters, config);
+  await initRuntimeVersion(runtimeParameters, customRuntimeParameters, config);
+  await initRuntimeStyles(runtimeParameters, config);
```

Comment:
These will execute sequentially, but it can be done in parallel with [one (annoying) tranformation](https://developers.google.com/web/fundamentals/primers/async-functions).